doc/rst/usingchapel/QUICKSTART.rst

```diff
@@ -162,7 +162,7 @@ What's next?
 For more information about Chapel, refer to the following resources:
 
 ============================ ==================================================
-Online documentation:        :ref:`chapel.cray.com/docs <chapel-documentation>`
+Online documentation:        :ref:`chapel-lang.org/docs <chapel-documentation>`
```

Comment:
Should this be https://chapel-lang...?

Reply:
I'm not certain.  I figured that the rst->html scripts must do something magical here (given that there wasn't an https:// before), but I didn't look into it.  Where's @ben-albrecht when we need him?  Are you suggesting it from a "in order to make it work" perspective or a "wouldn't it be clearer if..." point of view?

Reply:
I was being consistent with the previous inconsistency.  :)